MangoAPI.Client/src/app/components/chats/chats.component.html

```diff
@@ -327,7 +363,47 @@ <h5>Chats</h5>
           </li>
         </ul>
       </div>
-
+      <div *ngIf="_replyStateService.reply" class="reply">
+        <!-- Uploaded to: SVG Repo, www.svgrepo.com, Generator: SVG Repo Mixer Tools -->
```

Comment:
Clear this comment please

Reply:
it doesn't work without the comment

Reply:
> it doesn't work without the comment

Are you kidding me? XD 

So, it is like Linus Torvalds' comments, without them Linux kernel will not work :D